scripts/generate_certs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# make get-certs
+kubectl -n flotta get secrets flotta-ca --template="{{index .data \"ca.crt\" | base64decode}}" >${test_dir}/${DEVICE_ID}_ca.pem
```

Comment:
Agree, it would be easier to sort the generated files.
I will move the certs into ${test_dir}/certs, the registration logs into ${test_dir}/registration, and so on got get_updates and heartbeat